src/utilities/validation/object-validator.test.ts

```diff
@@ -0,0 +1,382 @@
+import { ObjectValidator } from "./object-validator";
+import { ObjectValidationResult } from "../../types/object-validation-result";
+import { CollectionUtils } from "../../utilities/collection-utils";
+import { StringUtils } from "../../utilities/string-utils";
+
+interface TestObject {

```

Comment:
If you have some time, I would consider:

1. Pulling in the `TestResource` interface from `andculturecode-javascript-testing` and updating these tests to use it

2. Wrapping the stub validator implementation in a region